acq_stat_reports/config.py

```diff
@@ -0,0 +1,21 @@
+class Config:
+    # "Top-level data directory",
+    data_dir = "."
+
+    # "Default figure width in inches",
+    figure_width = 5
+
+    # "Default figure height in inches",
+    figure_height = 2.5
+
+    # "Close matplotlib figures after plotting",
+    close_figures = False
+
+    # "Do not include bad stars in the reports",
+    remove_bad_stars = False
```

Comment:
bad stars will make things look worse than they are. To me, the real question is why should we include bad stars?

Reply:
The question of removing bad stars depends critically on the question you are asking, or in the case of this app, which plot you are making. 

If the question is "How accurate is the acq. prob model assuming we know the star magnitudes?" then we want to remove bad stars. That's what I do in all the model calibration and validation work. Eventually I'd like to see a plot in the acq_stats_report that looks like one of the validation plots that we look at for model promotion, with `remove_bad_stars=True`.

If the question is "In flight statistics, are we acquiring as many stars as we expect from the model?" then we should not remove bad stars. That is because encountering bad stars is an unavoidable fact of flight operations. If 50% of stars were bad then that would have a big impact on the answer to this question and we need to know that.

So that would imply that removing bad stars is an attribute of each plot (where a plot answers a question). The most common will be `remove_bad_stars=False`, but I think it would be best to be explicit about this and not define a default.

Reply:
I removed this option in PR #9